lab-14-chris/test/server-test.js

```diff
@@ -0,0 +1,286 @@
+'use strict';
+
+const server = require('../server');
+const chai = require('chai');
+const http = require('chai-http');
+const expect = chai.expect;
+// const Promise = require('bluebird');
+// const fs = Promise.promisifyAll(require('fs'), {suffix: 'Prom'});
+
+chai.use(http);
+
+describe('Server method - Test', () => {
+  let app;
```

Comment:
you could flesh this out even more by doing a before block with a post, and have that as your mockObject being passed through your tests.